### PR TITLE
Bug/19723 revert sass root variable change

### PR DESCRIPTION
**Pull request by GuyHarwood**
Turns out we don't need the variable that allows us to prefix the root of linked assets.

any referenced assets are relative to the root of the CSS file anyway.

### CODE REVIEW

File: admin-assets/nginx/default.conf [hunk 1]
```diff
@@ -75,4 +77,4 @@ server {
 
   # prevents file downloads from automatically opening in older versions of IE https://helmetjs.github.io/docs/ienoopen/
   add_header X-Download-Options noopen;
-}
+}
```
Comment (GeorgeChatzigiannis) [inline admin-assets/nginx/default.conf]: EOF

File: admin/gulpfile.js [hunk 2]
```diff
@@ -72,9 +72,9 @@ gulp.task('build', ['sass', 'bundleJs', 'copyImages', 'copyPDFs', 'copyCsvFiles'
 
 gulp.task('sass', function () {
   return gulp.src('./assets/**/*.scss')
-    .pipe(sassVariables({
+/*     .pipe(sassVariables({
```
Comment (GeorgeChatzigiannis) [inline admin/gulpfile.js]: why not remove it completely for now? can always revert it through git

Comment (jon-shipley) [reply admin/gulpfile.js]: yes - please remove if it's not in use.

Comment (GuyHarwood) [reply admin/gulpfile.js]: @GeorgeChatzigiannis @jon-shipley yep, it should be gone.  commented temporarily to see if all worked ok without it.

**Pull request merged**